README.md

```diff
@@ -51,49 +57,83 @@ There are 4 files at the location: src/config
 - bridge_relayer_config_dev.toml
 - bridge_relayer_config.toml
 
-1. edit bridge_relayer_config_dev.toml and bridge_relayer_config.toml
-2. set smart contract's address and genesis block values
```

Comment:
Suggestion: deployment block (to avoid confusion with the usual meaning of the genesis block)